src/ui/Root.fs

```diff
@@ -14,6 +16,8 @@ let Root () =
         router.onUrlChanged setUrl
         router.children [
             match url with
+            | [] -> Home()

```

Comment:
Do we want to consider adding additional routes to home?

Reply:
Probably not.